.gitignore

```diff
@@ -1,7 +1,7 @@
 # Custom
 
 # Remove comment for the line below before you push your code to GitHub!
-# .env
+.env
```

Comment:
Don't delete the comment, otherwise it wouldn't upload my example `.env` file

Reply:
Done